### ISSUE TITLE
NGIAB throwing error while running with AWI_002

### ISSUE THREAD

**Issue by arpita0911patel** [CIROH-UA/NGIAB-CloudInfra#87]
## Current behavior
Seeing below error while trying to run the NGIAB using latest image: 

At declaration of smc_profile size, soil_reservoir.n_soil_layers = 0
terminate called after throwing an instance of ‘realization::ConfigurationException’
 what(): Multi BMI formulation cannot be created from config: cannot find available data provider to satisfy set of deferred provisions for nested module at index 1: {ice_fraction_xinanjiang}
./HelloNGEN.sh: line 116:  14 Aborted         $run_command

## Expected behavior
Run successfully

## Steps to replicate behavior (include URLs)

1. Run NGIAB using AWI_002 data.

**Comment by arpita0911patel** [comment]
Nels Frazier
  [3:08 PM](https://cirohworkspace.slack.com/archives/C06E5A1JPKQ/p1705612106898349)
The realization needs updated. The latest NGEN submodules include updates to the models where some names have changed

**Comment by benlee0423** [comment]
Do we need to create AWI_003 with new realization?

**Comment by arpita0911patel** [comment]
Yes, waiting on @hellkite500 to provide the updated realization file as I see many differences between the two and would like to make sure we use the correct one.

**Comment by benlee0423** [comment]
Attached is new realization file, and tested with latest image, and it worked.
[awi_simplified_realization.json](https://github.com/CIROH-UA/NGIAB-CloudInfra/files/14016519/awi_simplified_realization.json)

Changes I made

```
"sloth_ice_fraction_xinan(1,double,1,node)": "0.0",
--> "sloth_ice_fraction_xinanjiang(1,double,1,node)": "0.0",

"ice_fraction_xinan": "sloth_ice_fraction_xinan",
--> "ice_fraction_xinanjiang" : "sloth_ice_fraction_xinanjiang", 
```

Run Result:                   

```
./guide.sh

...
...

Finished routing
NGen top-level timings:
	NGen::init: 7.8695
	NGen::simulation: 14.4193
	NGen::routing: 101.238
************ TIMING SUMMARY ************
----------------------------------------
Network graph construction: 0.64 secs, 0.63 %
Forcing array construction: 95.06 secs, 93.91 %
Routing computations: 1.52 secs, 1.5 %
Output writing: 4.0 secs, 3.96 %
----------------------------------------
Total execution time: 101.22 secs

real	2m4.881s
user	2m15.961s
sys	0m16.752s
Finished executing command successfully.

```

**Comment by benlee0423** [comment]
@hellkite500 
Please confirm if the change above in the realization file looks correct to you.

**Comment by hellkite500** [comment]
Yes, this is the minimal change to get the latest CFE to work correctly via the model engine.  Thanks for trying this out!
